static/common.css

```diff
@@ -170,10 +170,29 @@ article.link span.actuallink {
 }
 
 /* Discussion */
+
+p.annotation{
+    font-size: 0.95rem
+}
+
 .comment {
     display: flex;
 }
 
+.Comment--TextArea{
```

Comment:
I just think we should be consistent. So either `.comment--textarea` or rename everything else to be CamelCase.

But this looks like BEM, which I am all for—but [in BEM docs](http://getbem.com/introduction/), it's all lowercase and actually, `--` is for states. 

So IMHO it should be `.comment`, `.comment__textarea` and `.comment_submit`. 

And yeah, we should decide and migrate the rest as well :)

Reply:
Redone by BEM. More in slack.